.github/CODEOWNERS

```diff
@@ -19,3 +19,7 @@ lib/ClangImporter           @zoecarver @hyp @egorzhdan
 lib/PrintAsClang            @zoecarver @hyp @egorzhdan
 stdlib/public/Cxx           @zoecarver @hyp @egorzhdan
 test/Interop                @zoecarver @hyp @egorzhdan
+
+# Threading implementation
+include/swift/Threading @al45tair
+lib/Threading           @al45tair
```

Comment:
Could we add a leading `/` just to be on the safe side? And a trailing `/` to visually differentiate between directories and files in patterns? The latter would make it easier for humans to parse this file as it grows, I believe.

Reply:
Won't that break path matching? I imagine there was a reason not to add leading and trailing slashes for paths not modified in this PR.

Reply:
Fair enough. No, it won't break the matching. I think people just went with the less verbose pattern intuitively.